CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Added
 * **dso-toolkit:** Smoke test voor `dso-toolkit` SCSS compile ([#1168](https://github.com/dso-toolkit/dso-toolkit/issues/1168))
+* **dso-toolkit + styling:** Link lists ([#1159](https://github.com/dso-toolkit/dso-toolkit/issues/1159))
```

Comment:
`styling` package bestaat niet meer, is nu `sources`

Reply:
aangepast